apps/web/components/examples/blockchain/lesson-progress-card.tsx

```
@@ -0,0 +1,301 @@
+              </div>
+            ) : displayCompleted ? (
+              <div className="w-12 h-12 rounded-full bg-green-100 flex items-center justify-center">
+                <CheckCircleIconSolid className="w-8 h-8 text-green-600" />
+              </div>
+            ) : (
+              <button
+                onClick={handleCompleteLesson}
+                disabled={isLoading}
+                className="w-12 h-12 rounded-full bg-blue-600 hover:bg-blue-700 disabled:bg-blue-400 flex items-center justify-center transition-colors"
+              >
+                {isLoading ? (
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)